shared/translations/i18n.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+import i18n from "i18next";
+import { initReactI18next } from "react-i18next";
+
+import de_DE from "./de_DE.json";
+import en_US from "./default.json";
+import pt_PT from "./pt_PT.json";
+
+const outlineTranslation = {
```

Comment:
can we just export the `init` method here?

Reply:
yes, done